File: src/Models/CellarLocation.java
package Models;

public class CellarLocation implements HasID{
    public int location_id;
    public String section;
    public int rack_number;
    public int bottle_position;

    public CellarLocation(int location_id, String section, int rack_number, int bottle_position) {
        this.location_id = location_id;
        this.section = section;
        this.rack_number = rack_number;
        this.bottle_position = bottle_position;
    }

    public int getID() {
        return location_id;
    }

    public void setLocation_id(int location_id) {
        this.location_id = location_id;
    }

    public String getSection() {
        return section;
    }

    public void setSection(String section) {
        this.section = section;
    }

    public int getRack_number() {
        return rack_number;
    }

    public void setRack_number(int rack_number) {
        this.rack_number = rack_number;
    }

    public int getBottle_position() {
        return bottle_position;
    }

    public void setBottle_position(int bottle_position) {
        this.bottle_position = bottle_position;
    }

    @Override
    public String toString() {
        return "CellarLocation{" +
                "location_id=" + location_id +
                ", section='" + section + '\'' +
                ", rack_number='" + rack_number + '\'' +
                ", bottle_position='" + bottle_position + '\'' +
                '}';
    }
}

File: Python code/Entities/CellarLocation.py

class CellarLocation:
    def __init__(self, location_id, section, rack_number, bottle_position):
        self.location_id = location_id
        self.section = section
        self.rack_number = rack_number
        self.bottle_position = bottle_position

    def get_location_id(self):
        return self.location_id

    def get_section(self):
        return self.section

    def get_rack_number(self):
        return self.rack_number

    def get_bottle_position(self):
        return self.bottle_position

    def set_location_id(self, location_id):
        self.location_id = location_id

    def set_section(self, section):
        self.section = section

    def set_rack_number(self, rack_number):
        self.rack_number = rack_number

    def set_bottle_position(self, bottle_position):
        self.bottle_position = bottle_position

    def __str__(self):
        return f"Location: {self.section}-{self.rack_number}-{self.bottle_position}"